### DISCUSSION TITLE
Embedded Boundaries: Is `flag.isRegular` the same as `volfrac==1`?

### DISCUSSION THREAD

**Discussion by etpalmer63** [AMReX-Codes/amrex#2798] [Q&A] [answered]
A user asks:
>When using embedded boundaries (EB), is `flag.isRegular()` the same
>as `volfrac==1`?

**Comment by WeiqunZhang** [comment, marked as answer]
No, the two are different.  There could be covered cells that have a volume fraction of 1 and a completely covered face.  A regular cell has a volume fraction of 1 and all its faces have an area fraction of 1.
